storm-core/src/clj/backtype/storm/command/dev_zookeeper.clj

```diff
@@ -14,13 +14,15 @@
 ;; See the License for the specific language governing permissions and
 ;; limitations under the License.
 (ns backtype.storm.command.dev-zookeeper
-  (:use [backtype.storm zookeeper util config])
+  (:require [backtype.storm.zookeeper :as zoo]
+            [backtype.storm.config :as config])
+  (:use [backtype.storm util ])
```

Comment:
Trailing space after util.

Is there a reason we're keeping util in a use form instead of putting it in a require, unlike the others?

The second item in the jira is:
'2. Order imports use, require, import.'
but here you've put the use after the require.